orbit/Containerfile

```diff
@@ -18,10 +18,6 @@ RUN python3 -m venv /radius-venv && \
 COPY . /orbit
 WORKDIR /orbit
 
-RUN mkdir -p /var/orbit/ && \
```

Comment:
need to keep the mkdir right so that it doesn't fail when the parent dir doesn't exist, or will it just create them automatically?

Reply:
I don't recall if I thought about this. I'll add it back in